docs/build/http/api.md

```diff
@@ -1,14 +1,15 @@
 # HTTP API
 
-The HTTP API allows you to manually make HTTP requests that create, modify, and query streams on a remote Ceramic node. If you are building an application, you will usually interact with a node using one of the [Ceramic clients](../build/javascript/installation.md), however this documentation is useful if:
+The HTTP API allows you to manually make REST HTTP requests to write, query, and pin streams on a remote Ceramic node. If you are building an application, you will usually interact with Ceramic using a client, such as the [**JS HTTP Client**](../javascript/installation.md#js-http-client) or the [**JS Core Client**](../javascript/installation.md#js-core-client). 
 
-- You have a special use case where you directly want to use HTTP requests
+However the HTTP API is useful if:
+
+- You have a special use case where you directly want to make manual HTTP requests
 - You want to implement an HTTP client in a new language
 
-!!! info "Gateway mode"
-    Not all of the API methods are available if the Ceramic node runs in
-    *gateway mode*. This option disables writes, which is useful when exposing your node to
-    the internet. **Methods disabled in gateway mode will be clearly marked.**
+!!! warning "Gateway mode"
+    
+    Not all of the HTTP API methods are available if the Ceramic node runs in *gateway mode*. This option disables writes, which is useful when exposing your node to the internet. **Methods disabled in gateway mode will be clearly marked.**
```

Comment:
Maybe mention gateway.ceramic.network here as an example of a node that runs in gateway mode only?

Reply:
Just added a link to the tools/hosted-nodes/community-nodes.md#gateways